src/components/footer/Footer.css

```diff
@@ -289,6 +294,10 @@ footer {
         text-align: center;
     }
 
+    .brand-container{
```

Comment:
Sorry, I just realized I accidentally duplicated the codes from lines 297 to 300.
(they are the same as lines 233-236)